tests/VolleyManagement.Specs/TeamsContext/ManageTeamPlayersSteps.cs

```diff
@@ -1,87 +1,153 @@
 ﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using FluentAssertions;
 using TechTalk.SpecFlow;
+using VolleyManagement.Contracts;
+using VolleyManagement.Data.MsSql.Entities;
+using VolleyManagement.Domain.TeamsAggregate;
+using VolleyManagement.Specs.Infrastructure;
+using VolleyManagement.Specs.Infrastructure.IOC;
 
 namespace VolleyManagement.Specs.TeamsContext
 {
     [Binding]
+    [Scope(Feature = "Manage Team Players")]
     public class ManageTeamPlayersSteps
     {
-        [Given(@"Team A exists")]
-        public void GivenTeamAExists()
-        {
-            ScenarioContext.Current.Pending();
-        }
-        
-        [Given(@"I have added Jane Doe as a team player")]
-        public void GivenIHaveAddedJaneDoeAsATeamPlayer()
-        {
-            ScenarioContext.Current.Pending();
-        }
-        
-        [Given(@"Ivan Ivanov is a team player")]
-        public void GivenIvanIvanovIsATeamPlayer()
+        private readonly ITeamService _teamService;
+        private TeamId _teamId;
+        private PlayerId _captainId;
+        private readonly List<PlayerId> _playerToAdd;
+        private readonly ICollection<PlayerId> _playersToRemove;
+        private Exception _exception;
+
+        public ManageTeamPlayersSteps()
         {
-            ScenarioContext.Current.Pending();
+            _teamService = IocProvider.Get<ITeamService>();
+            _playerToAdd = new List<PlayerId>();
+            _playersToRemove = new List<PlayerId>();
         }
-        
-        [Given(@"I have added John Smith as a team player")]
-        public void GivenIHaveAddedJohnSmithAsATeamPlayer()
+
+        [Given(@"Team (.*) exists")]
+        public void GivenTeamExists(string name)
         {
-            ScenarioContext.Current.Pending();
+            var team = TestDbAdapter.CreateTeamWithCaptain(name, "First", "Last");
```

Comment:
Make fName:teamName, lName:"Captain" so it is unique. In case we create few teams in scope of single scenario